dbms/src/Server/Server.cpp

```diff
@@ -265,7 +289,12 @@ struct TiFlashProxyConfig
                 args_map[engine_store_address] = config.getString("flash.service_addr");
             else
                 args_map[engine_store_advertise_address] = args_map[engine_store_address];
-            args_map[engine_label] = engine_label_value;
+
+            auto disaggregated_mode = getDisaggregatedMode(config);
+            if (disaggregated_mode == DisaggregatedMode::Compute)
+                args_map[engine_label] = DISAGGREGATED_MODE_COMPUTE;
```

Comment:
After we can push Selection to tiflash_storage node, we need check if this node `isDisaggregatedStorageMode()`.
Because tiflash_storage node doesn't need `FilterBlockInputStream`.

Reply:
Used in `getDisaggregatedMode()` for validation.

After we can push Selection to tiflash_storage node, we need check if this node isDisaggregatedStorageMode().
Because tiflash_storage node doesn't need FilterBlockInputStream, but non-disaggregated mode and tiflash_compute mode need FilterBlockInputStream.

Reply:
add `getProxyLabelByDisaggregatedMode()`